hops/featurestore.py

```diff
@@ -1314,7 +1314,12 @@ def import_featuregroup_s3(storage_connector, featuregroup, path=None, primary_k
     arguments = locals()
     arguments['type'] = "S3"
     core._do_import_featuregroup(json.dumps(arguments))
-    job.launch_job(featuregroup)
+    #path to json file in hdfs
+    input_json_path = '--job_spec hdfs:///Projects/' + \
```

Comment:
@SirOibaf Is it ok to have `hdfs` scheme hard-coded? We are not using hopsfs in Hopsworks if I'm not mistaken

Reply:
Yes that can work.